### PR TITLE
Fixed images sourced from tars and IDs to have final filesystem

### LINKED ISSUE #11: Make images sourced from IDs and tars convert to final filesystem
Currently, their filesystem still comprises of layer folders

### PR DESCRIPTION
**Pull request by cftorres**
fixes #11 
@aaron-prindle @nkubala @abbytiz

### CODE REVIEW

**Comment by abbytiz** [comment]
lgtm

**Comment by abbytiz** [comment]
LGTM

File: utils/docker_utils.go [hunk 1]
```diff
@@ -303,3 +308,59 @@ func getImageConfig(image string) (container.Config, error) {
...
+	manifest := filepath.Join(tempLayerDir, "manifest.json")
+	layers, err := getLayersFromManifest(manifest)
+	if err != nil {
+		return err
+	}
+
+	for _, layer := range layers {
+		layerTar := filepath.Join(tempLayerDir, layer)
+		if _, err := os.Stat(layerTar); err != nil {
+			glog.Infof("Did not unpack layer %s because no layer.tar found", layer)
```
Comment (nkubala) [inline utils/docker_utils.go]: Why would a layer not have a layer.tar file?

Comment (cftorres) [inline utils/docker_utils.go]: If it's an empty layer it only has a manifest and a metadata json file, but no contents to unpack

Comment (nkubala) [inline utils/docker_utils.go]: ahh, makes sense

**Pull request merged**